fetch/src/2_process.r

```diff
@@ -0,0 +1,22 @@
+get_data <- function() {
+  # Get the data from ScienceBase
+  mendota_file <- file.path(project_output_dir, 'model_RMSEs.csv')
+  item_file_download('5d925066e4b0c4f70d0d0599', names = 'me_RMSE.csv', destinations = mendota_file, overwrite_file = TRUE)
+}
```

Comment:
This `get_data()` function seems to be redundant, since you're already downloading the data with `fetch_data()`

Reply:
Yes, that makes sense.